bindings/cs/rl.net/LiveModel.cs

```diff
@@ -868,6 +940,40 @@ public void QueueOutcomeEvent(string eventId, uint slotIndex, string outcomeJson
                 }
         }
 
+        public bool TryQueueOutcomeEvent(string eventId, string slotId, float outcome, ApiStatus apiStatus = null)
+        {
+            int result = LiveModelReportOutcomeSlotStringIdF(this.DangerousGetHandle(), eventId, slotId, outcome, apiStatus.ToNativeHandleOrNullptrDangerous());
+
+            GC.KeepAlive(this);
+            return result == NativeMethods.SuccessStatus;
+        }
+
+        public void QueueOutcomeEvent(string eventId, string slotId, float outcome)
+        {
+            using (ApiStatus apiStatus = new ApiStatus())
+                if (!this.TryQueueOutcomeEvent(eventId, slotId, outcome, apiStatus))
+                {
+                    throw new RLException(apiStatus);
+                }
+        }
+
+        public bool TryQueueOutcomeEvent(string eventId, string slotId, string outcomeJson, ApiStatus apiStatus = null)
+        {
+            int result = LiveModelReportOutcomeSlotStringIdJson(this.DangerousGetHandle(), eventId, slotId, outcomeJson, apiStatus.ToNativeHandleOrNullptrDangerous());
+
+            GC.KeepAlive(this);
+            return result == NativeMethods.SuccessStatus;
+        }
+
+        public void QueueOutcomeEvent(string eventId, string slotId, string outcomeJson)
+        {
+            using (ApiStatus apiStatus = new ApiStatus())
+                if (!this.TryQueueOutcomeEvent(eventId, slotId, outcomeJson, apiStatus))
```

Comment:
[Minor][Nit] Looks like there might be some whitespace/alignment issues here.

Reply:
fixed